site/content/en/docs/tasks/_index.md

```diff
@@ -63,6 +63,7 @@ As a platform developer, you can learn how to:
 - [Integrate a custom Job with Kueue](dev/integrate_a_custom_job).
 - [Enable pprof endpoints](dev/enabling_pprof_endpoints).
 - [Develop a custom AdmissionCheck Controller](dev/develop-acc).
+- [Integration a custom workload with existing frameworks](dev/external_frameworks)
```

Comment:
```suggestion
- [Integrate a custom workload with Kueue using built-in frameworks](dev/external_frameworks)
```
Also, move it up just below "Integrate a custom Job with Kueue.

Reply:
done.